renderlib/graphics/RenderGLPT.cpp

```diff
@@ -206,7 +206,6 @@ RenderGLPT::doRender(const CCamera& camera)
     uint32_t c0, c1, c2, c3;
     m_scene->getFirst4EnabledChannels(c0, c1, c2, c3);
     m_imgGpu.updateVolumeData4x16(m_scene->m_volume.get(), c0, c1, c2, c3);
-    m_imgGpu.updateLutGPU(m_scene->m_volume.get(), c0, c1, c2, c3, m_scene->m_material);
```

Comment:
Why was this line removed?

Reply:
Because it's in the part of the code that is only supposed to update the volume data.  It was the wrong place to update the LUT on GPU and because of the dirty flags, it is redundant to have it happen here.